common/src/main/java/org/apache/uniffle/common/util/ByteBufUtils.java

```diff
@@ -24,7 +24,10 @@
 public class ByteBufUtils {
 
   public static int encodedLength(String s) {
-    return 4 + s.getBytes(StandardCharsets.UTF_8).length;
+    if (s == null) {
```

Comment:
We can use `StringUtils.isEmpty(s)`.

Reply:
`Null`and `an empty string` have different meanings, here we need to judge whether it is null.